src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
+
+    client.evaluate(`${inputString}.toString()`, { frameId }).then(res => {
```

Comment:
the `toString` worries me :)

Reply:
Not sure how to do it differently if we have e.g. a function and we expect the body of it to be displayed.